src/generateSdk/sdkGenerator/PythonSdkGenerator.ts

```diff
@@ -231,13 +235,31 @@ class SdkGenerator implements SdkGeneratorInterface {
     return "Any";
   }
 
+  generateEnum(e: Enum): string {
+    const enumType = e.cases[0].type;
+    const allTypesAreTheSame = e.cases.map((v) => v.type).every((type) => type === enumType);
+
+    if (!allTypesAreTheSame) {
+      throw new Error("All enum cases must be the same type. Fix enum " + e.name + " and try again.");
+    }
+
+    switch (enumType) {
+      case AstNodeType.StringLiteral:
+        return `class ${e.name}(StrEnum):\n\t${e.cases.map((e: EnumCase, i: number) => `${e.name} = "${e.value}"`).join("\n\t")}`;
```

Comment:
Instead of `StrEnum` which is only available from Pyhon 3.11, I should use:

```
from enum import Enum

class MyEnum(str, Enum):
    state1 = 'state1'
    state2 = 'state2'
```